src/components/ProcessCreate/StepForm/Censuses.tsx

```diff
@@ -44,15 +44,13 @@ export const TokenCensus = () => (
 )
 
 const Csp = lazy(() => {
-  if (import.meta.env.features._census.csp) {
-    return import('./CensusCsp').then((module) => ({ default: module.StepFormCensusCsp }))
-  }
-
-  return Promise.resolve({ default: () => <></> })
+  return import('./CensusCsp').then((module) => ({ default: module.StepFormCensusCsp }))
 })
 
-export const CspCensus = () => (
-  <SuspenseLoader>
-    <Csp />
-  </SuspenseLoader>
+export const CspCensusGithub = () => (
+  <SuspenseLoader>{import.meta.env.features._census.csp_github && <Csp provider='github' />}</SuspenseLoader>
+)
+
+export const CspCensusGoogle = () => (
+  <SuspenseLoader>{import.meta.env.features._census.csp_google && <Csp provider='google' />}</SuspenseLoader>
```

Comment:
This does not work for the treeshake... Sadly it seems the conditionals can only be defined as in the components above (in a clear `if` statement condition). Ternary statements, at least inside a react component view, do not seem to be working.